R/badge.R

```diff
@@ -41,18 +44,49 @@ NULL
 #' @rdname badges
 #' @export
 use_badge <- function(badge_name, href, src) {
-  if (has_badge(href)) {
+  link <- glue("[![{badge_name}]({src})]({href})")
+  badge_lines <- badge_lines()
```

Comment:
It feels odd to get the line numbers back separate from the README file path and to then re-find the README. But I'm not sure what to do about that.

Reply:
I couldn't figure out how to factor it out without making the fallback code (when there's no README, or no badge comments) more complicated. Someone can take another stab at it in #517